docs/spec/deprecated-schema-v1.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Update deprecated schema v1 images
+description: Update deprecated schema v1 images
+keywords: registry, on-prem, images, tags, repository, distribution, api, advanced, manifest
+---
+
```

Comment:
I notice we only set the `title` in the metadata, but not as an actual heading in Markdown. While that (probably) work when rendering this page on docs.docker.com, it doesn't show up on GitHub, so we may want to add a Markdown header here;

```suggestion

## Update deprecated image manifest version 2, schema 1 images to schema 2
```